From bc5e60892570cf488b292c486b7a3d3d979430c9 Mon Sep 17 00:00:00 2001
From: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Date: Wed, 5 Sep 2018 16:13:03 +0530
Subject: [PATCH 097/909] clk: renesas: r8a7796: Remove INTC-AP and INTC-EX
 clocks

commit eb2b15bdbc4670893b4e1b4c4faa3c94a6d75511 from
git://git.kernel.org/pub/scm/linux/kernel/git/horms/renesas-bsp.git

According to the R-Car Gen3 Hardware Manual Rev 1.00, the the module clock
status, module clock control, and reset control bits of the INTC-AP and
INTC-EX modules on R-Car M3 are removed.

Continue to change the clock properties of the gic and irqc nodes in the
device tree.

Signed-off-by: Takeshi Kihara <takeshi.kihara.df@renesas.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/clk/renesas/r8a7796-cpg-mssr.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/drivers/clk/renesas/r8a7796-cpg-mssr.c b/drivers/clk/renesas/r8a7796-cpg-mssr.c
index 7e98fb8b1074..f2a0c188ddfe 100644
--- a/drivers/clk/renesas/r8a7796-cpg-mssr.c
+++ b/drivers/clk/renesas/r8a7796-cpg-mssr.c
@@ -146,8 +146,6 @@ static const struct mssr_mod_clk r8a7796_mod_clks[] __initconst = {
 	DEF_MOD("usb-dmac0",		 330,	R8A7796_CLK_S3D1),
 	DEF_MOD("usb-dmac1",		 331,	R8A7796_CLK_S3D1),
 	DEF_MOD("rwdt",			 402,	R8A7796_CLK_R),
-	DEF_MOD("intc-ex",		 407,	R8A7796_CLK_CP),
-	DEF_MOD("intc-ap",		 408,	R8A7796_CLK_S0D3),
 	DEF_MOD("audmac1",		 501,	R8A7796_CLK_S0D3),
 	DEF_MOD("audmac0",		 502,	R8A7796_CLK_S0D3),
 	DEF_MOD("drif7",		 508,	R8A7796_CLK_S3D2),
@@ -246,11 +244,6 @@ static const struct mssr_mod_clk r8a7796_mod_clks[] __initconst = {
 	DEF_MOD("scu-src0",		1031,	MOD_CLK_ID(1017)),
 };
 
-static const unsigned int r8a7796_crit_mod_clks[] __initconst = {
-	MOD_CLK_ID(408),	/* INTC-AP (GIC) */
-};
-
-
 /*
  * CPG Clock Data
  */
@@ -332,10 +325,6 @@ const struct cpg_mssr_info r8a7796_cpg_mssr_info __initconst = {
 	.num_mod_clks = ARRAY_SIZE(r8a7796_mod_clks),
 	.num_hw_mod_clks = 12 * 32,
 
-	/* Critical Module Clocks */
-	.crit_mod_clks = r8a7796_crit_mod_clks,
-	.num_crit_mod_clks = ARRAY_SIZE(r8a7796_crit_mod_clks),
-
 	/* Callbacks */
 	.init = r8a7796_cpg_mssr_init,
 	.cpg_clk_register = rcar_gen3_cpg_clk_register,
-- 
2.17.1

